grails-app/migrations/0.8.x/changelog-2021-07-08-1530-add-hold-location-type.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+  <property name="now" value="now()"/>
+  <changeSet author="dhatala" id="0807202115300-0">
+    <preConditions onFail="MARK_RAN">
+      <sqlCheck expectedResult="0">
+        select count(*) from location_type where location_type_code = "INTERNAL_LOCATION";
```

Comment:
Let's just call this HOLD_LOCATION throughout. Unless I'm missing something, I think the check should be where id = "HOLD_LOCATION".